### PR TITLE
LPD-26789 LPD-26926 Add test for aggregation term parameters using object relationship field

### PR DESCRIPTION
**Pull request by carolmariaabb**
https://liferay.atlassian.net/browse/LPD-26789
https://liferay.atlassian.net/browse/LPD-26926

### CODE REVIEW

File: modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java [hunk 1]
```diff
@@ -2706,53 +2706,92 @@ public void testGetObjectEntriesWithAccountEntryRestricted3()
...
-			dtoConverterContext, _objectDefinition2,
+				_objectRelationshipFieldName, parentObjectEntry2.getId()));
+
+		ObjectEntry childObjectEntry = _addObjectEntry(
+			_objectDefinition2,
 			Collections.singletonMap(
-				_objectRelationshipFieldName, objectEntry2.getId()));
+				_objectRelationshipFieldName, parentObjectEntry2.getId()));
+
+		_addRoleUser(new String[] {ActionKeys.VIEW}, _objectDefinition2, _user);
```
Comment (marcelabc) [inline modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java:2749]: Question: Does this test need this?

Comment (marcelabc) [reply modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java:2749]: I saw why we need this line.

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**